### PR TITLE
[PAN-2943] Renames various eea methods to priv methods, with associated docs

### PR DESCRIPTION
**Pull request by josh-richardson**
## PR description
Renames various eea methods to priv methods, as discussed on call this morning.

### CODE REVIEW

**Comment by josh-richardson** [comment]
I hope this commit addresses the required changes: https://github.com/PegaSysEng/pantheon/pull/1736/commits/5bc6242bb385c113885160786424802a613138c9

**Comment by lucassaldanha** [comment]
> Enabling of the priv_ methods using --rpc-http-api and --ws-http-api needs updating to add PRIV. 
Do we want to have the PRIV group or will we enable everything with the EEA group?

**Comment by MadelineMurray** [comment]
As discussed, EEA enables EEA methods and PRIV enables the extended Pantheon privacy

**Comment by josh-richardson** [comment]
My latest commit addresses the two new refactors. Unsure whether to put `tech.pegasys.pantheon.tests.acceptance.dsl.transaction.eea.PrivGetTransactionCountTransaction` into its own package now, or not.

**Comment by lucassaldanha** [comment]
> Unsure whether to put `tech.pegasys.pantheon.tests.acceptance.dsl.transaction.eea.PrivGetTransactionCountTransaction` into its own package now, or not.

I'd put it into a `tech.pegasys.pantheon.tests.acceptance.dsl.transaction.priv` package.

File: ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcMethodsFactory.java [hunk 1]
```diff
@@ -322,6 +322,7 @@
           new AdminChangeLogLevel(parameter));
     }
     if (rpcApis.contains(RpcApis.EEA)) {
+      //      TODO: josh-richardson change this
```
Comment (rain-on) [inline ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcMethodsFactory.java]: remove the todo?

Comment (CjHare) [inline ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/JsonRpcMethodsFactory.java]: Why is there still a TODO? ...shouldn't this have been done before committing to master?

**Pull request merged**